docs/content/preview/releases/yba-releases/v2.18.md

```diff
@@ -21,6 +21,59 @@ What follows are the release notes for all releases in the **YugabyteDB Anywhere
+* Upgrades netty-codec, fixing many vulnerabilities and enhancing security across older branches. Also backports play3.0 and overrides netty-codec 4.1.89 with an updated version. PLAT-11238
+* Allows deletion of pexlock even when owned by a different user, preventing errors and ensuring a smoother operation. The lock file location is now alterable to avoid future issues. PLAT-12445
+* Fixes an issue where the resize node operation was failing for AWS due to a typo in the update_disk method. The issue was preventing the creation of a universe with an existing nvme1 mount. PLAT-12645
+* Enables idempotent `mv` command usage across file systems during replicated migration, addressing failure issue when root install directory exists on a different file system. Also resolves minor logging issues. PLAT-12836
+* Preserves the node-agent folder during instance cleanup for on-premises nodes. PLAT-12896
+* Enables successful pause/universe function even when disks are not mounted by UUID by correctly utilizing the host_info parameter. PLAT-12911
+* Updates both `-classic` and `-core` Java packages in your home library to version 1.2.11 for more advanced capability. PLAT-11662
+* Shifts to Play 3.0.x, leveraging Pekko to mitigate vulnerabilities in the latest Akka library. PLAT-11680
+* Allows usage of user-provided `skipProvisioning` value instead of defaulting to `false` when editing the access key, retaining the manual provisioning settings in `edit provider`. PLAT-12393
+* Corrects the migration of boolean and integer config values from replicated to YBA installer, ensuring they are written as their correct types, not strings. This fix passes the preflight validation checks for subsequent upgrades. PLAT-12407,PLAT-12406
+* Fixes unintended behavior where YCQL table-by-table backup was incorrectly backing up all tables for each run. PLAT-12460
+* Alters order of node-agent installation and node provisioning to ensure yugabyte user is set up for node agent in on-premises situations. PLAT-12533
+* Automatically updates the pexvenv to sync with new PYTHON_EXECUTABLE after an OS or direct Python upgrade. PLAT-12618
+* Disables SSH key expiry alert for existing customers, ensuring a smooth user experience. PLAT-12724
+* Renames the create provider config label from "Kubernetes" to specific terms including "VMware Tanzu", "Red Hat OpenShift", and "Managed Kubernetes Service". PLAT-10888
+* Updates yugabyted-ui packages /x/net and /x/crypto for enhanced security. Backports available upgrades to all stable versions. Also migrates to Play 3.0.x. PLAT-11660
+* Redesigns the delete region modal to eliminate fields specific to AWS, reducing confusion when deleting regions across different cloud providers. PLAT-12446
+* Enables use of HTTPS to gather platform metrics when TLS is turned on, improving security in data collection. PLAT-12210
```

Comment:
improvement?

Reply:
no thats a bug.. it used HTTP instead of HTTPS. ill add it